.github/ISSUE_TEMPLATE/security-vulnerability.md

```diff
@@ -6,7 +6,6 @@ about: Report vulnerability in the project
 <!-- Thank you for your contribution. Before you submit the issue:
 1. Search open and closed issues for duplicates.
 2. Read the contributing guidelines.
-3. In case of vulnerabilities of CVSS Base Score *7.0* or above, send an email directly to kyma-security@googlegroups.com instead of using this issue tracker. You can also use email for vulnerabilities with lower severity if you prefer to keep your report confidential.
```

Comment:
Have you clarified if we need such an email address by ourself?

Reply:
I clarified it with @hardl, and reverted that change for the time being.